serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -85,49 +85,19 @@ Knative CLI::
 +
 --
 .Create a project using Knative CLI
-[source,shell]
+[source,shell,subs="attributes"]
 ----
-kn workflow create --name my-project --extension quarkus-jsonp,quarkus-smallrye-openapi
+kn workflow create \
```

Comment:
The command with kn plugin version 0.21.0 fails with
```
Error: unknown flag: --kogito-version
```

Reply:
With 0.21.2 it fails when trying to use productized RHBQ, because it defaults to use quarkus-maven-plugin with groupId `io.quarkus.platform`, while for productized releases it should be `com.redhat.quarkus.platform`.
Seems like another bug in CLI.

Reply:
reported as https://issues.redhat.com/browse/KOGITO-7740